test/e2e/webdriver/driver.js

```diff
@@ -777,14 +777,27 @@ class Driver {
     const response = await this.driver.get(`${this.extensionUrl}/${page}.html`);
     // Wait for asynchronous JavaScript to load
     if (waitForControllers) {
-      await this.driver.wait(
-        until.elementLocated(this.buildLocator('.controller-loaded')),
-        10 * 1000,
-      );
```

Comment:
just abstracted this into a function, so we can use re-use it in other places (like the ui test)